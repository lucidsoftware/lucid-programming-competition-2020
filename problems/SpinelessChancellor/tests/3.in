4
DEAD
BEEF
WAS1
HERE
128
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 yes
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF no
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF yes
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF yes
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF yes
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD yes
BEEF yes
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD no
BEEF yes
WAS1 yes
HERE no
DEAD yes
BEEF no
WAS1 no
HERE yes
DEAD no
BEEF yes
WAS1 yes
HERE no
DEAD yes
BEEF unsure
WAS1 no
HERE yes
DEAD no
BEEF unsure
WAS1 yes
HERE no
DEAD yes
BEEF unsure
WAS1 no
HERE yes
DEAD no
BEEF unsure
WAS1 yes
HERE no
DEAD yes
BEEF unsure
WAS1 no
HERE yes
DEAD no
BEEF unsure
WAS1 yes
HERE no
DEAD yes
BEEF unsure
WAS1 no
HERE yes
DEAD no
BEEF unsure
WAS1 yes
HERE no
DEAD yes
BEEF unsure
WAS1 no
HERE yes
DEAD no
BEEF unsure
WAS1 yes
HERE no
DEAD yes
BEEF unsure
WAS1 no
HERE yes
DEAD no
BEEF unsure
WAS1 yes
HERE no
DEAD yes
BEEF unsure
WAS1 no
HERE yes
DEAD no
BEEF unsure
WAS1 yes
HERE no
DEAD yes
BEEF unsure
WAS1 no
HERE yes
DEAD no
BEEF unsure
WAS1 yes
HERE unsure
DEAD yes
BEEF unsure
WAS1 no
HERE unsure
