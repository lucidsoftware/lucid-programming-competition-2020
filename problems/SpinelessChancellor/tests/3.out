DEAD yes
BEEF no
WAS1 yes
HERE abstain
